src/mjolnir/valhalla_ingest_transit.cc

```diff
@@ -0,0 +1,22 @@
+#include "baldr/rapidjson_utils.h"
+#include "mjolnir/ingest_transit.h"
+
+int main(int argc, char** argv) {
+  if (argc < 2) {
+    std::cerr << "Usage: " << std::string(argv[0]) << " Error" << std::endl;
+    return 1;
+  }
```

Comment:
right now I'd have no idea how this exe works before I look into the source, on error we don't print any usage. can do it in a follow-up PR as well, but we really shouldn't leave it like this.. have a look here for copy/paste (almost):
https://github.com/valhalla/valhalla/blob/gtfs-fetch-transit/src/valhalla_export_edges.cc/#L143-L185

Reply:
yeah we just copied this from fetch transit  which doesnt use nice args its standard old c style stuff